RecoParticleFlow/PFProducer/python/particleFlowBlock_cff.py

```diff
@@ -9,17 +9,15 @@
 # include "RecoTracker/TrackProducer/data/CTFFinalFitWithMaterial.cff"
 from RecoParticleFlow.PFProducer.particleFlowBlock_cfi import *
 
-_phase2_hgcal_Importers = deepcopy(particleFlowBlock.elementImporters)
+from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
 # kill tracks in the HGCal
-_phase2_hgcal_Importers[5].importerName = cms.string('GeneralTracksImporterWithVeto')
-_phase2_hgcal_Importers[5].veto = cms.InputTag('hgcalTrackCollection:TracksInHGCal')
-#_phase2_hgcal_Importers[2].source_ee = cms.InputTag('particleFlowSuperClusterHGCal')
-#_phase2_hgcal_Importers.append(
-#    cms.PSet( importerName = cms.string("HGCalClusterImporter"),
-#              source = cms.InputTag("particleFlowClusterHGCal"),
-#              BCtoPFCMap = cms.InputTag('particleFlowSuperClusterHGCal:PFClusterAssociationEBEE') ),
-#)
-_phase2_hgcal_Linkers = deepcopy(particleFlowBlock.linkDefinitions)
+phase2_hgcal.toModify(
+    particleFlowBlock,
+    elementImporters = { 5 : dict(
```

Comment:
I'm not completely happy with the new interface, but I'm unable to come up with anything better. I would prefer (slightly) the proposed syntax over  the "clone+modify+clone+modify+etc" chain.

By the way, does the implementation already support
```
some_era.toModify(foo, bar={1: None})
```
to delete an item? I don't know if there is actually need for it, so the only argument for it would be consistency.